### ISSUE TITLE
jdeps --multi-release --print-module-deps fails

### ISSUE THREAD

**Issue by smoell** [corretto/corretto-17#56]
### Describe the bug
The bug is already described in detail here: https://bugs.openjdk.java.net/browse/JDK-8277165.

In my specific case: 

```
jdeps --multi-release 17 --print-module-deps -cp target/BOOT-INF/lib/*.jar target/CustomerService-0.0.1.jar
Exception in thread "main" java.lang.Error: java.util.concurrent.ExecutionException: com.sun.tools.jdeps.MultiReleaseException
	at jdk.jdeps/com.sun.tools.jdeps.DependencyFinder.waitForTasksCompleted(DependencyFinder.java:271)
	at jdk.jdeps/com.sun.tools.jdeps.DependencyFinder.parse(DependencyFinder.java:133)
	at jdk.jdeps/com.sun.tools.jdeps.DepsAnalyzer.run(DepsAnalyzer.java:129)
	at jdk.jdeps/com.sun.tools.jdeps.ModuleExportsAnalyzer.run(ModuleExportsAnalyzer.java:74)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask$ListModuleDeps.run(JdepsTask.java:1047)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask.run(JdepsTask.java:574)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask.run(JdepsTask.java:533)
	at jdk.jdeps/com.sun.tools.jdeps.Main.main(Main.java:49)
Caused by: java.util.concurrent.ExecutionException: com.sun.tools.jdeps.MultiReleaseException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
	at jdk.jdeps/com.sun.tools.jdeps.DependencyFinder.waitForTasksCompleted(DependencyFinder.java:267)
	... 7 more
Caused by: com.sun.tools.jdeps.MultiReleaseException
	at jdk.jdeps/com.sun.tools.jdeps.VersionHelper.add(VersionHelper.java:62)
	at jdk.jdeps/com.sun.tools.jdeps.ClassFileReader$JarFileReader.readClassFile(ClassFileReader.java:360)
	at jdk.jdeps/com.sun.tools.jdeps.ClassFileReader$JarFileIterator.hasNext(ClassFileReader.java:402)
	at jdk.jdeps/com.sun.tools.jdeps.DependencyFinder.lambda$parse$5(DependencyFinder.java:179)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

### To Reproduce
Already described here: https://bugs.openjdk.java.net/browse/JDK-8277165

### Expected behavior
jdeps should run without an exception and print out the correct list of module dependencies. Please backport OpenJDK 18 fix.

### Platform information
    OS: Mac OS X and Amazon Linux 2
    Version: Corretto-17.0.2.8.1

**Comment by lutkerd** [comment]
Thank you for letting us know. The JDK ticket already has a request to backport the fix to JDK17 and it should make it into the next quarterly release in mid April.

**Comment by olivergillespie** [comment]
Actually, it looks like this fix is not in the April release and instead is scheduled for 17.0.4 in July - sorry for the wait.

**Comment by smoell** [comment]
No worries, I'll switch to Corretto 18 temporarily.

**Comment by lutkerd** [comment]
The fix is in the 17.0.4 release that went out last week.
